pkg/virtctl/expose/expose.go

```diff
@@ -69,6 +70,7 @@ virtualmachineinstance (vmi), virtualmachine (vm), virtualmachineinstancereplica
 	cmd.Flags().StringVar(&strTargetPort, "target-port", "", "Name or number for the port on the VM that the service should direct traffic to. Optional.")
 	cmd.Flags().StringVar(&strServiceType, "type", "ClusterIP", "Type for this service: ClusterIP, NodePort, or LoadBalancer.")
 	cmd.Flags().StringVar(&portName, "port-name", "", "Name of the port. Optional.")
+	cmd.Flags().StringVar(&strIPFamily, "ip-family", "ipv4", "IP family over which the service will be exposed.")
```

Comment:
Add the valid values for example capitalized like IPv4 are not working.

Reply:
this is just the default; despite it being used also in the documentation, since we lower case it farther down the code, it won't matter much.

If you insist, I'll do it.